docs/integrations/sources/s3.md

```diff
@@ -209,6 +209,7 @@ The Jsonl parser uses pyarrow hence,only the line-delimited JSON format is suppo
 
 | Version | Date       | Pull Request                                                                                                    | Subject                                                                                    |
 |:--------|:-----------|:----------------------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------|
+| 1.0.3   | 2023-03-06 | [23195](https://github.com/airbytehq/airbyte/pull/23195)                                                        | Fix datetime format string                                                                 |
```

Comment:
@olivermeyer this PR is one file change away from being ready to merge - You also need to bump the version in the [Dockerfile](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-s3/Dockerfile).

Reply:
Done :)